cmake/FindFFTW3.cmake

```diff
@@ -1,21 +1,51 @@
 CHECK_REQUIRED_LIB(FFTW3F fftw3f fftw3.h libfftw3f-3 "")
 CHECK_REQUIRED_LIB(FFTW3D fftw3 fftw3.h libfftw3-3  "")
+if(NOT WIN32)
+	CHECK_REQUIRED_LIB(FFTW3F_THREADS fftw3f_threads fftw3.h "" "")
```

Comment:
Hello Tunay,

I guess this line needs to have fftw3f.h instead of fftw3.h?

best,
Markus

Reply:
> fftw3f.h

There is no `fftw3f.h` as far as I can tell.